CHANGELOG.md

```diff
@@ -0,0 +1,36 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/) and this
+project adheres to [Semantic Versioning](http://semver.org/).
+
+## [v0.3.0] - 2019-07-11
```

Comment:
If we drop the existing outputs then this will be a breaking change.

Reply:
Added those back since there's no real reason they need to be removed right now.

It may be a good idea to backlog that these should be removed at some point in the future as they're redundant.